### PR TITLE
Add Zendesk reload API support

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by lukas-slovak** [comment]
LGTM

File: gooddata-java-model/src/main/java/com/gooddata/sdk/model/connector/Reload.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    public Reload(Map<String, Long> startTimes) {
+        this(null, startTimes, null, null, null);
+    }
+
+    @JsonCreator
+    public Reload(
+            @JsonProperty("id") final Integer id,
+            @JsonProperty("startTimes") final Map<String, Long> startTimes,
+            @JsonProperty("status") final ReloadStatus status,
```
Comment (liry) [inline gooddata-java-model/src/main/java/com/gooddata/sdk/model/connector/Reload.java:55]: Please see https://github.com/gooddata/gooddata-java/blob/master/CONTRIBUTING.md#enums

File: gooddata-java/src/main/java/com/gooddata/sdk/service/connector/ConnectorService.java [hunk 2]
```diff
@@ -260,6 +261,45 @@ public void updateSettings(final Project project, final Settings settings) {
...
+        isTrue(reload.getUri().isPresent(), "reload.uri");
+        return getZendesk4Reload(reload.getUri().get());
+    }
+
+    /**
+     * Get Zendesk reload.
+     * @param reloadUri existing reload URI
+     * @return reload
+     */
+    public Reload getZendesk4Reload(final String reloadUri) {
```
Comment (liry) [inline gooddata-java/src/main/java/com/gooddata/sdk/service/connector/ConnectorService.java:284]: Please rename to `getZendesk4ReloadByUri`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-java-model/src/test/java/com/gooddata/sdk/model/connector/ReloadTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        assertThat(reload.getProcessUri().get(), is(PROCESS_LINK));
+    }
+
+    @Test
+    public void testGetIntegrationUri() {
+        assertThat(reload.getIntegrationUri().get(), is(INTEGRATION_LINK));
+    }
+
+    @Test
+    public void constrictor2nd() {
```
Comment (liry) [inline gooddata-java-model/src/test/java/com/gooddata/sdk/model/connector/ReloadTest.java:102]: You meant "constructor"? :)

File: gooddata-java/src/main/java/com/gooddata/sdk/service/connector/ConnectorService.java (hunk 2 above)
Comment (sadam21) [reply gooddata-java/src/main/java/com/gooddata/sdk/service/connector/ConnectorService.java:284]: Done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-java-model/src/main/java/com/gooddata/sdk/model/connector/Reload.java (hunk 1 above)
Comment (sadam21) [reply gooddata-java-model/src/main/java/com/gooddata/sdk/model/connector/Reload.java]: Enum `ReloadStatus` replaced by `String`.

File: gooddata-java-model/src/test/java/com/gooddata/sdk/model/connector/ReloadTest.java (hunk 3 above)
Comment (sadam21) [reply gooddata-java-model/src/test/java/com/gooddata/sdk/model/connector/ReloadTest.java]: Fixed :-)

**Review by jimirocks — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-java-merge-compliance-check/185/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-java-fossa-scanning-zuul-el7/25/ : SUCCESS in 12s (non-voting)

**Pull request merged**